### PR TITLE
Migrate from LambdaControls to MidnightControls

### PR DESCRIPTION
**Pull request by Motschen**
Removes the LambdaControls integration in favor of MidnightControls integration.
MidnightControls is a continuation of LambdaControls.

### CODE REVIEW

File: src/main/java/io/github/foundationgames/automobility/Automobility.java [hunk 1]
```diff
@@ -37,7 +37,7 @@ public void onInitialize() {
 
         AutomobilityData.setup();
 
-        ControllerUtils.initLCHandler();
+        ControllerUtils.initMCHandler();
```
Comment (FoundationGames) [inline src/main/java/io/github/foundationgames/automobility/Automobility.java]: I think all instances of "MC" in relation to "MidnightControls" should be expanded out to "MidnightControls"

File: src/main/java/io/github/foundationgames/automobility/util/midnightcontrols/AutomobilityMidnightControls.java [hunk 2]
```diff
@@ -24,7 +25,7 @@
  * LambdaControls is discontinued and this has a nearly 100% chance of being removed in the future (once the mod is broken)
```
Comment (FoundationGames) [inline src/main/java/io/github/foundationgames/automobility/util/midnightcontrols/AutomobilityMidnightControls.java]: You can remove this comment if you'd like

**Review by FoundationGames — CHANGES_REQUESTED** [review]
Really awesome that you're continuing LambdaControls.

File: src/main/java/io/github/foundationgames/automobility/util/midnightcontrols/AutomobilityMidnightControls.java (hunk 2 above)
Comment (Motschen) [reply src/main/java/io/github/foundationgames/automobility/util/midnightcontrols/AutomobilityMidnightControls.java]: Done in the latest commit.

File: src/main/java/io/github/foundationgames/automobility/Automobility.java (hunk 1 above)
Comment (Motschen) [reply src/main/java/io/github/foundationgames/automobility/Automobility.java]: Yeah, you're right, I did so in the latest commit.

**Review by FoundationGames — APPROVED** [review]
Looks good, will merge ASAP

**Pull request merged**